src/components/general/NavigationDrawer/components/Child.tsx

```diff
@@ -1,10 +1,21 @@
-import React, { FC, useCallback, useMemo } from 'react';
+import React, { FC, useCallback, useEffect, useState } from 'react';
 import styles from './styles.less';
 import { NavigationComponentProps } from '..';
 import NavigationItem from './NavigationItem';
+import { useTooltipRef } from '@equinor/fusion-components';
 
 const Child: FC<NavigationComponentProps> = ({ navigationItem, onChange }) => {
     const { id, isActive, title, onClick } = navigationItem;
+    const textRef = React.useRef<HTMLElement | null>(null);
+    const [shouldHaveTooltip, setShouldHaveTooltip] = useState(false);
+    const tooltipRef = useTooltipRef(title, 'right');
+
+    useEffect(() => {
+        if (textRef.current) {
+            const isOverflowing = textRef.current.offsetWidth < textRef.current.scrollWidth;
+            setShouldHaveTooltip(isOverflowing);
+        }
+    }, [textRef])
```

Comment:
The dependency should probably be on textRef.current since the ref really doesn't change. Also, what if the title changes? Should there be a dependency on that as well?

Reply:
When adding textRef.current as a dependency the tooltip does not work. We have observed that with textRef the getNavigationContent is run when you hover the title, but with textRef.current it does not. Not sure why or if this is the reason that the tooltip is only shown with textRef as dependency. Suggestions?

Reply:
Seems strange 🤔 The ref itself shouldn't change, only the value of the `current` property. Therefore, having a dependency on the ref would only run the effect after initial render. In most cases that's enough since the `current` property is set in between rendering and the effect, but if the component is rerendered with a different DOM node, or the title changes the tooltip would not.

Reply:
Very strange indeed. Spent some time trying to debug without getting any further.

Reply:
Allright. As long as it works the way it is now, we'll merge it and figure it out later

Reply:
I tested some more to make sure, and unfortunately discovered that the section tooltip only works after it's child gets rendered. Group and child tooltips works as they should, but something weird is going on with section :/

Reply:
@GryNagel My latest commit seems to fix the issue with tooltip on sections

Reply:
Looks perfect! Thanks (and sorry I completely forgot about this PR)! 👏